charts/tekton/templates/tekton-engine.yaml

```diff
@@ -0,0 +1,2018 @@
+{{- if .Values.engine.enabled -}}
```

Comment:
Does it make sense to have everything else like tekton `triggers`, `dashboard`, `webhook` if engine would be disabled?

Currently I see from the `tekton-engine.yaml` file in vp-kubernetes-aws like we can separately create only engine, only dashboards, only triggers and only webhooks. But as I understand, it shouldn't make sense, cause we should enable dashboard and webhook only if we have a `tekton engine enabled`, right?

Reply:
Ok, I add additional bool check into triggers/dashboard to make sure that if engine is disabled - they are disabled as well.

Reply:
Pls review latest commit